src/lib/scanner/golbatScanner.js

```diff
@@ -0,0 +1,83 @@
+class GolbatScanner {
+	constructor(db) {
+		this.dbs = db
+	}
+
+	// eslint-disable-next-line camelcase
+	async getGymName(gym_id) {
+		try {
```

Comment:
should not be needed for golbat (is this null for mad? I can't remember what I did)
[although it could be I am wrong actually, I can't remember why gym name might be used rather than pokestop name -- I added the pokestop name to golbat's webhook I am sure; to avoid this useless db lookup]

Reply:
actually the golbatScanner.js is a 1:1 copy from rdmScanner.js to give the possibility for adjustments to golbat in the future

Reply:
removed this part from golbatSanner.js, works without it

Reply:
Changes reset as this will brake reactmap's poracle integration.

Reply:
Yes, I forgot about this